### ISSUE TITLE
bug(COMPONENT): Blank dropdown in version choices

### ISSUE THREAD

**Issue by ctyy** [angular/components#22659]
#### Reproduction

Use StackBlitz to reproduce your issue: 
- Components: https://stackblitz.com/fork/components-issue
- Harnesses: https://stackblitz.com/fork/harness-issue

Steps to reproduce:
1. visit https://material.angular.io/
2. Click on the version
 
 
#### Expected Behavior

What behavior were you expecting to see?
Version dropdown list show up

#### Actual Behavior

What behavior did you actually see?
Blank version dropdown.

#### Environment

- Angular:
- CDK/Material:
- Browser(s):
- Operating System (e.g. Windows, macOS, Ubuntu):

**Comment by Splaktar** [comment]
Duplicate of https://github.com/angular/material.angular.io/issues/976. This is fixed, just waiting on deployment of the new version. cc @mmalerba
